src/main/java/io/github/bookcrawler/controllers/DiscountFetchingController.java

```diff
@@ -1,28 +1,55 @@
 package io.github.bookcrawler.controllers;
 
+
+import io.github.bookcrawler.cache.DatabaseCacheForDifferentLibraries;
 import io.github.bookcrawler.core.DiscountFetchingService;
 import io.github.bookcrawler.entities.BookInfo;
+import io.github.bookcrawler.repositories.BookInfoRepository;
+import io.github.bookcrawler.utilities.Library;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Controller;
 import org.springframework.ui.Model;
 import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
 
 import java.io.IOException;
-import java.util.ArrayList;
-import java.util.Collection;
 import java.util.List;
 
 @Controller
 public class DiscountFetchingController {
 
-    @GetMapping("/")
-    public String hello(Model model) throws IOException {
-        List<BookInfo> list = new ArrayList<>();
+    @Autowired
+    DatabaseCacheForDifferentLibraries databaseCacheForDifferentLibraries;
 
-        Collection<BookInfo> bookInfos = new DiscountFetchingService().getAllBooks();
+    @Autowired
+    DiscountFetchingService discountFetchingService;
 
-        bookInfos.stream().forEach(list::add);
-        model.addAttribute("books", list);
+    @Autowired
+    BookInfoRepository bookInfoRepository;
 
+    @GetMapping("/")
+    public String hello() throws IOException {
```

Comment:
Is it something like Hello World ? :smile: I think greeting will be better name but I don't know your reason :smile: